app/controllers/quiz_response_maps_controller.rb

```
@@ -0,0 +1,108 @@
+    # multiple teams get the correct team's quiz rather than whatever find_by returns first.
+    if reviewee_team_id.present? && !reviewee_team_id.zero?
+      reviewee_team = Team.find_by(id: reviewee_team_id)
+    else
+      # Fall back: find the review map for this reviewer on this assignment. find_by returns
+      # only one record — if a student reviews multiple teams this is ambiguous, which is why
+      # callers should always supply reviewee_team_id.
+      reviewer_participant_lookup = AssignmentParticipant.find_by(user_id: reviewer_user_id, parent_id: assignment_id)
+      review_map = ReviewResponseMap.find_by(reviewer_id: reviewer_user_id, reviewed_object_id: assignment_id)
+      unless review_map
+        review_map = ReviewResponseMap.find_by(reviewer_id: reviewer_participant_lookup&.id, reviewed_object_id: assignment_id)
+      end
```

RuboCop (1.86.1)

[convention] Use the double pipe equals operator `||=` instead.

(Style/OrAssignment)